pkgs/development/libraries/pslib/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, cmake, pkgconfig, zlib, libpng, libjpeg, libungif, libtiff
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pslib";
+  version = "0.4.6";
+
+  src = fetchurl {
+    name = "${pname}-snixource-${version}.tar.gz";
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "0m191ckqj1kj2yvxiilqw26x4vrn7pnlc2vy636yphjxr02q8bk4";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ zlib libpng libjpeg libungif libtiff ];
+
+  doCheck = true;
+
+  outputs = [ "out" "dev" "doc" ];
+
+  installPhase = ''
+  mkdir -p $out/lib
+  for path in *.so *.so.* *.o *.o.*; do
+      mv $path $out/lib/
+  done
+  mkdir -p $dev/include
+  mv ../include/libps $dev/include
+  if test -d nix-support; then
+      mv nix-support $dev
+  fi
+  mkdir -p $doc/share/doc/${pname}
+  cp -r ../doc/. $doc/share/doc/${pname}
```

Comment:
can you indent this a bit more? (2 spaces)

Reply:
Sure. Thank you for reminding.